.github/workflows/desktop.yml

```diff
@@ -9,26 +9,34 @@ env:
 
 jobs:
   build:
-    name: ${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.architecture }}-${{ matrix.python_version }}
+    name: ${{ matrix.os }}-${{ matrix.build_type }}-${{ matrix.architecture }}-${{ matrix.msvc_runtime}}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
         build_type: ["Release", "Debug"]
-        architecture: ["x64",]
+        architecture: ["x64", "x86"]
+        msvc_runtime: ["static", "dynamic"]
         python_version: [3.7]
         include:
         - os: windows-latest
-          architecture: "x64"
-          vcpkg_triplet: "x64-windows-static"
-          msbuild_platform: "x64"
+          vcpkg_triplet_suffix: "windows-static"
+        - os: windows-latest
+          msvc_runtime: dynamic
```

Comment:
should dynamic be in quotes for consistency?

Reply:
Looks like it's still not in quotes here.

Reply:
Sorry to have missed this. Fixed in the latest commit.